CHANGELOG.md

```diff
@@ -31,10 +37,6 @@
 - Bump Flipper to v0.143.0
 - Bump kotlin to v1.6.21
```

Comment:
This change is made after the release. Let's move it to the `Next Release` section.